### ISSUE TITLE
Need to update jekyll version for ruby 2.7.0.

### ISSUE THREAD

**Issue by HuvarVer** [AbsaOSS/hermes#42]
## Describe the bug
There is a warning `Using the last argument as keyword parameters is deprecated` during generating gh-pages with Ruby 2.7.0 by jekyll.

Depends on issue https://github.com/jekyll/jekyll/issues/7947

## Desktop (please complete the following information):
 - Ruby 2.7.0
 - Jekyll 3.8.5

**Comment by leimao** [comment]
Any follow-up on this? Thank you.

**Comment by Zejnilovic** [comment]
> Any follow-up on this? Thank you.

Hello @leimao, can I ask what sparked your interest in this issue? Why do you ask?

I am asking because this is something that we have as a low priority since our GitHub Pages work since GH Pages are using Ruby 2.5.8 and Jekyll 3.8.7.

**Comment by leimao** [comment]
> > Any follow-up on this? Thank you.
> 
> Hello @leimao, can I ask what sparked your interest in this issue? Why do you ask?
> 
> I am asking because this is something that we have as a low priority since our GitHub Pages work since GH Pages are using Ruby 2.5.8 and Jekyll 3.8.7.

Sorry, I thought hermes is something used by the Jekyll which caused this warning. But I was wrong. Essentially every Jekyll based website was affected by this. However, upgrading Jekyll to 4.1.0 seems to have solved the problem.

**Comment by leimao** [comment]
See https://github.com/jekyll/jekyll/issues/7947
